bin/sandbox

```diff
@@ -58,6 +58,7 @@ gem 'solidus_backend'
 gem 'solidus_sample'
 gem '$extension_name', path: '..'
 gem 'solidus_auth_devise'
+gem 'solidus_i18n'
```

Comment:
This file is generated/updated with the `solidus extension .` command if I recall correctly. If we run it again in the future it will try to remove this line and we'll probably forget that having it here was intentional. What about adding a small comment to explain that this is actually needed?

Reply:
I just noticed that `solidus_i18n` is already present in [solidus_dev_support](https://github.com/solidusio/solidus_dev_support/blob/master/lib/solidus_dev_support/templates/extension/bin/sandbox.tt#L55) so there shouldn't be any problem if we run that command again in the future. However, I'm not sure if I need to also add `rails_i18n` 🤔 . WDYT?